Demo/Demo/Features/CardDemoViewController.swift

```diff
@@ -0,0 +1,105 @@
+import UIKit
+
+class CardDemoViewController: FeatureBaseViewController {
+    // TODO: Validate card fields are filled out
+    // TODO: Add proper spacing / styling / max characters to card fields
+    // TODO: Animate pay button to indicate loading
+    // TODO: Disable pay button until fields are filled out
+
+    private enum Constants {
+        static let stackViewSpacing: CGFloat = 16
+    }
```

Comment:
Do we need this enum, or can we just have a single property.

Reply:
We had a few other properties being repeated so added some additional properties to this enum